lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,186 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
+        'use strict';
+        describe('edx.commerce.ReceiptView', function() {
+            var data, courseResponseData, providerResponseData = null;
+
+            var createReceiptView = function() {
+                return new ReceiptView({el: $('#receipt-container')});
+            };
+
+            var loadReceiptFixture = function() {
+                var receiptFixture = readFixtures('templates/commerce/receipt.underscore');
+                var providerFixture = readFixtures('templates/commerce/provider.underscore');
```

Comment:
There should only be 1 var declaration at the top of the function, comma separating the variable names.